### PR TITLE
Support py37

### PR DESCRIPTION
**Pull request by madhulika95b**
@ldko @somexpert Added python 3.7 to the testing environment, Removed django 1.10 version. Please review the changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,36 +10,36 @@ different criteria such as scope, author, year, etc.
...
+* Python 2.7, 3.5- 3.7
 
 
 Installation
 ------------
 
 1.  Download and install from source code.
     ```sh
-	$ pip install git+git://github.com/unt-libraries/django-citeit.git
+	$  pip install git+git://github.com/unt-libraries/django-citeit.git
```
Comment (somexpert) [inline README.md]: I think this extra space isn't needed.

File: README.md [hunk 2]
```diff
@@ -10,36 +10,36 @@ different criteria such as scope, author, year, etc.
...
+	    urlpatterns = [
             url(r'^admin/', admin.site.urls),
             url(r'^withers/', include('citeIt.urls'))
         ]
     ```
 
 4.  Migrate/sync the database
     ```sh
-	$ python manage.py migrate
+	$  python manage.py migrate
```
Comment (ldko) [inline README.md]: I don't think any of these spacing changes are needed. It makes the spacing inconsistent in raw form and with what we usually do, and we are not using the same IDEs, so basing spacing on a specific IDE is not ideal.

File: README.md (hunk 1 above)
Comment (madhulika95b) [reply README.md]: Well, I added it 'cos the first letters were missing in the rendered part of README in my IDE. And adding it didn't change the rendering in GitLab, so I thought adding it wouldn't hurt.

File: README.md (hunk 2 above)
Comment (madhulika95b) [reply README.md]: Ummm...ok. If it is working fine in all your IDEs except just mine, I can revert these spaces.

**Pull request merged**